contrib/completion/zsh/_docker-compose

```diff
@@ -52,7 +52,8 @@ __docker-compose_services_with_key() {
         | sed -n -e '/^services:/,/^[^ ]/p' \
         | sed -n 's/^  //p' \
         | awk '/^[a-zA-Z0-9]/{printf "\n"};{printf $0;next;}' \
-        | awk -F: -v key=": +$1:" '$0 ~ key {print $1}' \
+        | grep " \+$1:" \
```

Comment:
@albers I'm dropping the first `:` on purpose :)
With the extra `:`, the autocomplete only works if the docker-compose file is ordered in a specific way.

For instance, while this service will be matched (assuming `$1` is `build`):
```yaml
my-app:
  build: ./my-dir
  ports:
    - 80
```
…this one won't:
```yaml
my-app:
  ports:
    - 80
  build: ./my-dir
```